tests/conftest.py

```diff
@@ -29,30 +32,59 @@
     )
 
 
-# A fixture only used for testing mlflow-trace package integration.
-@pytest.fixture(autouse=IS_TRACING_SDK_ONLY)
-def remote_backend_for_tracing_sdk_test(monkeypatch):
-    # Check remote backend is running or not
+@pytest.fixture(autouse=IS_TRACING_SDK_ONLY, scope="session")
+def remote_backend_for_tracing_sdk_test():
+    """
+    A fixture to start a remote backend for testing mlflow-tracing package integration.
+    Since the tracing SDK has to be tested in an environment that has minimal dependencies,
+    we need to start a tracking backend in an isolated uv environment.
+    """
+    # Use fixed port for simplicity
+    port = 5000
```

Comment:
https://github.com/mlflow/mlflow/blob/0992ea9f03c021c2fb081a90b3ac0fdac9d0d950/tests/helper_functions.py#L44

can we use `get_safe_port`? This test would fail when there is a zombie tracking server running on port 5000.